patroni/postgresql.py

```diff
@@ -665,7 +665,9 @@ def create_replica(self, clone_member):
                     break
             # if the method is basebackup, then use the built-in
             if replica_method == "basebackup":
-                ret = self.basebackup(connstring, env)
+                base_backup_options = (self.config[replica_method].copy()
+                                       if self.config.get(replica_method, {}) else {})
+                ret = self.basebackup(connstring, env, base_backup_options)
```

Comment:
It could be just a single line:
```python
                ret = self.basebackup(connstring, env, (self.config.get(replica_method) or {})).copy())
```

Or may be two two line, but in any case `if ... else` construction is not really useful here.